site/content/docs/main/resource-filtering.md

```diff
@@ -101,6 +101,14 @@ Includes cluster-scoped resources. Cannot work with `--include-cluster-scoped-re
 
 For more information read the [Kubernetes label selector documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors)
```

Comment:
Looking this docs, couldn't we achieve the same result using the `--selector` flag with `in`?

Ex.: Do these commands produce the same result? 
- `velero backup create nginx-backup --selector "app in (nginx,my-nginx,nginx-example)"`
- `velero backup create nginx-backup --or-selector "app=nginx|app=my-nginx|app=nginx-example"`

Did not try every possibility, but maybe this PR can be achieve only adding examples to the docs with different scenarios (and updating help description of  `--selector` flag in CLI)

Reply:
But I think the usage from the docs is limited. For example, if I have 2 (or more) different labels (KEYs), I can't do an OR with them ("Similarly the comma separator acts as an AND operator.")

Reply:
@mateusoliveira43 yes -- that's the sort of scenario the or selector was added for:
`velero backup create nginx-backup --or-selector "app=nginx|foo=bar"`

Reply:
If resources to be backed up are labeled either `env=prod` or `environment=production`, then we cannot use `--selector env=prod,environment=production`. But we can use `--or-selector "env=prod|environment=production"`

I have updated the PR to include examples and explanations.

Reply:
I have modified the syntax a little bit. The ` or ` keyword can be used in place of `|` pipe symbol.